torchaudio/functional.py

```diff
@@ -458,14 +458,17 @@ def phase_vocoder(
     factor of ``rate``.
 
     Args:
-        complex_specgrams (Tensor): Dimension of `(..., freq, time, complex=2)`
+        complex_specgrams (Tensor): Either a real tensor of dimension of `(..., freq, time, complex=2)`
+            or a tensor of dimension `(..., freq, time)` with complex dtype.
         rate (float): Speed-up factor
         phase_advance (Tensor): Expected phase advance in each bin. Dimension of (freq, 1)
 
     Returns:
-        Tensor: Complex Specgrams Stretch with dimension of `(..., freq, ceil(time/rate), complex=2)`
+        Tensor: Complex Specgrams Stretch with either a real dtype and dimension of
+                `(..., freq, ceil(time/rate), complex=2)` or
+                a complex dtype and dimension of `(..., freq, ceil(time/rate))`.
 
-    Example
+    (Old API) Example
```

Comment:
`Example` block gets a special treatment in documentation, so adding the differentiation after `Example`, like `Example - foo` gives better rendering.

<img width="845" alt="Screen Shot 2020-08-06 at 16 05 08" src="https://user-images.githubusercontent.com/855818/89577453-b4034680-d7fe-11ea-9699-146c569417e1.png">

vs 

<img width="846" alt="Screen Shot 2020-08-06 at 16 05 17" src="https://user-images.githubusercontent.com/855818/89577464-b6fe3700-d7fe-11ea-826a-fd9b8cf344b9.png">

Reply:
that's a good point! updated